packages/react-native-dogfood/src/components/VideoWrapper.tsx

```diff
@@ -36,6 +37,8 @@ export const VideoWrapper = ({ children }: PropsWithChildren<{}>) => {
     },
   });
 
+  usePushRegisterEffect(videoClient);
```

Comment:
usePushRegisterEffect must be called by integrator if and only if they want push notifications and as they initiate the client.

Some integrators may not use push notification since we ask for payment for it

Reply:
So its basically an optional call.. but if you call this hook and don't have the optional dependencies setup we throw an error. Making the development life clear and easy fixing potential

Reply:
With v6 chat we can introduce a similar hook for chat.. just an idea

Reply:
actually on second thoughts, we can do this.. we can check if the push config is present and then call the hook..

So basically if push config is set -> push is enabled in SDK else it is not enabled

will make a commit for that